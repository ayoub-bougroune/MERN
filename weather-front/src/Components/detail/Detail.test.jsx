import React from 'react';
import renderer from 'react-test-renderer';
import Detail from './index';
import {Route, MemoryRouter } from "react-router-dom";

test('The Detail renders correctly', () => {
  const detail = renderer.create(
      <MemoryRouter initialEntries={["detail/2553604"]}>
        <Route path='detail/:id'>
          <Detail />
        </Route>
      </MemoryRouter>
    );
  
  
  const tree = detail.toJSON();
  expect(tree).toMatchSnapshot();
});