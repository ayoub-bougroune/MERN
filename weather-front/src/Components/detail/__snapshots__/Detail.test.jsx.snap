// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Detail renders correctly 1`] = `null`;
